tests/test_update_sensor.py

```
@@ -981,31 +1841,26 @@ async def test_update_location_attributes_sets_locations(
     mock_hass: MagicMock,
     mock_config_entry: MockConfigEntry,
     sensor: MockSensor,
     method_name: str,
-    expected_m_attr: str,
-    expected_mi_attr: str,
+    expected_distance_attr: str,
 ) -> None:
-    """Parametrized test for distance calculation methods to validate m and mi attributes are set appropriately."""
+    """Distance calculation methods should set their native meter attribute."""
     updater = PlacesUpdater(mock_hass, mock_config_entry, sensor)
     sensor.is_attr_blank.side_effect = lambda k: False
```

Ruff (0.16.1)

[warning] Unused lambda argument: `k`

(ARG005)